src/dbt/kipptaf/models/extracts/tableau/rpt_tableau__schoolmint_grow_observation_details.sql

```diff
@@ -154,126 +155,337 @@ with
+            s.manager,
+            s.worker_original_hire_date,
+            s.assignment_status,
+
+            o.observation_id,
+            o.teacher_id,
+            o.form_long_name,
+            o.created,
+            o.observed_at,
+            o.observer_name,
+            o.overall_score,
+            o.glows,
+            o.grows,
+            o.score_measurement_id,
+            o.score_percentage,
+            o.row_score_value,
+            o.measurement_name,
+            o.measurement_scale_min,
+            o.measurement_scale_max,
+            o.tier,
+            o.text_box,
+
+            row_number() over (
+                partition by
+                    s.form_type,
+                    s.form_short_name,
+                    s.internal_id,
+                    o.score_measurement_id
+                order by o.observed_at desc
+            ) as rn_submission,
+
+        from scaffold as s
+        left join
+            observations as o
+            on cast(o.observed_at as date) between s.start_date and s.end_date
+            /* matches only on type and date for weekly forms */
+            and s.form_type in ('WT', 'O3')
+            and s.form_type = o.form_type
+            and s.user_id = o.teacher_id
+    ),
+
+    historical_overall_scores as (
+        select
+            employee_number,
+            academic_year,
+            pm_term as form_term,
+            etr_score,
+            so_score,
+            overall_score,
+            etr_tier,
+            so_tier,
+            overall_tier as tier,
+        from
+            {{
+                source(
+                    "performance_management",
+                    "src_performance_management__scores_overall_archive",
+                )
+            }}
+
+        union distinct
+
+        select
+            employee_number,
+            academic_year,
+            form_term,
+            null as etr_score,
+            null as so_score,
+            overall_score,
+            null as etr_tier,
+            null as so_tier,
+            tier,
+        from observation_details
+        where form_type = 'PM' and overall_score is not null
+    ),
+
+    historical_detail_scores as (
+        select
+            subject_employee_number as employee_number,
+            academic_year,
+            case
+                when pm_term = 'PM1'
+                then 'BOY'
+                when pm_term = 'PM2'
+                then 'MOY'
+                when pm_term = 'PM3'
+                then 'EOY'
+            end as form_term,
+            score_type,
+            observer_employee_number,
+            null as observer_name,
+            observed_at,
+            measurement_name,
+            score_value as row_score_value,
+
+        from
+            {{
+                source(
+                    "performance_management",
+                    "src_performance_management__scores_detail_archive",
+                )
+            }}
```

Comment:
create staging model for this and do transformations there

Reply:
damn i changed the transformations on this and its not on this commit, i took out this case statement